Source/Core/Common/StringUtil.h

```diff
@@ -58,6 +58,11 @@ bool TryParse(const std::string& str, T* output)
 {
   char* end_ptr = nullptr;
 
+  if (str == "true")
+    return 1;
+  else if (str == "false")
+    return 0;
```

Comment:
This function isn't supposed to return the parsed value, it's supposed to return whether the parse succeeded. The parsed value needs to be stored in the `output` parameter.

Also, the code you added only works when the string is all lowercase, which is a problem since Dolphin uses `True` and `False` when writing to INI files. It should be case insensitive to match `TryParse(const std::string&, bool*)`.

Reply:
Oops. I couldn't test since I was developing over ssh, will fix.